src/tex/sections/human_subset_selection.tex

```diff
@@ -2,4 +2,4 @@
 
 Several design choices were made when selecting a human subset. Firstly, from the full dataset, we keep all the questions that are labeled as advanced. In this way, we can obtain a deeper insight about the capabilities of \glspl{llm} on advanced tasks
 when compared to actual chemists. Secondly, we sample a maximum of three questions across all possible combinations of categories (i.e., knowledge or reasoning) and topics (e.g., organic chemistry, physical chemistry). Thirdly, we do not include any intuition questions
-in this subset. In total, \variable{output/num_human_answered_questions.txt} questions have been sampled for the human study. Then, this set is divided into two subsets based on the aforementioned combinations. One of the question subsets allows tool use, and the other does not.
+in this subset, because the correct answers are subjective, and would hinder an objective evaluation of the chemists against \glspl{llm}. In total, \variable{output/num_human_answered_questions.txt} questions have been sampled for the human study. Then, this set is divided into two subsets based on the aforementioned combinations. One of the question subsets allows tool use, and the other does not.
```

Comment:
thanks for including this argument. However, does this argument not raise the question of why we include preference data in the first place?

Reply:
i do not think so. In a way this argument is to say that the answers for these questions are defined by preference. 
while we want the LLM to align with the human preference, we do not want any preference questions when comparing the LLMs to humans due to the subjectivity.

 and also, the preference questions cannot have a human baseline per say, since they are defined by humans.

Reply:
I believe the interpretation of the "tiny subset" depends on its intended use. If it's solely for the human study, then the argument is valid—this was, in fact, our original motivation.

However, we also propose the "tiny subset" as a resource for quick and inexpensive evaluation of LLMs. From this perspective, we need to explain why the preference data does not provide a useful signal in this context.

Reply:
the preference data cannot have a human baseline right? since I feel it is a human baseline itself :D

Reply:
Yes - this was the argument why we excluded it.

However, we also propose this subset to be used for fast evaluation of LLMs - irrespective of any human baseline. For this usecase, arguments based on a human baseline are tricky or at least indirect.

Reply:
should I then change the argument to:

"The intutition questions represent a human baseline from a different study. We intend that our "tiny" subset is ued for a fast and fair evaluation of LLMs, independent of any human baselines."

Reply:
updated the argument